tests/unit/hook_packages/terraform/test_prepare_hook.py

```diff
@@ -696,37 +696,41 @@ def test_translate_to_cfn_empty(self):
         ]
 
         for tf_json in tf_jsons:
-            translated_cfn_dict = _translate_to_cfn(tf_json)
+            translated_cfn_dict = _translate_to_cfn(tf_json, "/output/dir", "/project/root")
```

Comment:
nit: some of the strings here could be constants/variables of this test class

Reply:
resolved